test/index.js

```diff
@@ -256,4 +256,63 @@ describe('trample', function() {
 
     assert.equal(res['foo_bar'], 'baz');
   });
+  it('should flatten object, keep the date value intact', function() {
+    var date_val = new Date();
+    var obj = {
+      'foobar123': {
+        'cat': 123,
+      },
+      'date': date_val
+    };
+
+    var res = trample(obj);
+    assert.equal(res['foobar123.cat'], 123);
+    assert.equal(res['date'], date_val);
+  });
```

Comment:
Suggestion regarding the test description. Also, changed `snake_case` to `camelCase`.

```suggestion
  });
  
  it('should keep date objects intact when flattening objects', function() {
    var dateVal = new Date();
    var obj = {
      foobar123: {
        cat: 123,
      },
      date: dateVal
    };

    var res = trample(obj);
    assert.equal(res['foobar123.cat'], 123);
    assert.equal(res['date'], dateVal);
  });
```